### PR TITLE
Solution to mini-challenge-3

### PR DESCRIPTION
**Pull request by seunmeme**
## Kindly help review my code.

### CODE REVIEW

File: src/binary-reversal/index.js [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+
+  let binaryValue = Number(value).toString(2);
+
+  while (binaryValue.length < 8) {
+    binaryValue = '0' + binaryValue;
+  }
```
Comment (bondz) [inline src/binary-reversal/index.js:12]: This is essentially `binaryValue.padStart(8, 0)`

File: src/list-sorting/index.js [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+      for(let j=0; j<haystack[i].length; j++){
+        if(haystack[i][j] === needle){
+          arrayIndex = i;
+          index = j;
+        }
+      }
+    }
+    output.push(arrayIndex, index);
+    return index !== -1 ? output : index;
+  }else{
```
Comment (bondz) [inline src/list-sorting/index.js:23]: Missed an opportunity to use `Early Return` from last week. You could drop this `else`

File: src/list-sorting/index.js [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+  }
+
+  if(typeof haystack[0] !== "number"){
+    output = [];
+    let index = -1;
+    let arrayIndex;
+
+    for(let i=0; i<haystack.length; i++){
+      for(let j=0; j<haystack[i].length; j++){
+        if(haystack[i][j] === needle){
```
Comment (bondz) [inline src/list-sorting/index.js:15]: See `[].lastIndexOf(...)`

File: src/list-sorting/index.js (hunk 2 above)
Comment (seunmeme) [reply src/list-sorting/index.js:23]: 😮 It will be implemented going forward.

File: src/list-sorting/index.js (hunk 3 above)
Comment (seunmeme) [reply src/list-sorting/index.js:15]: 🤔 I'll read more on it.

**Pull request merged**